CMakeLists.txt

```diff
@@ -33,6 +33,20 @@ install(EXPORT ArborXTargets
   DESTINATION lib/cmake/ArborX
 )
 
+SET(ARBORX_VERSION_STRING "0.9 (dev)")
+SET(ARBORX_GIT_COMMIT_HASH "No hash available")
+IF(EXISTS ${PROJECT_SOURCE_DIR}/.git)
+    FIND_PACKAGE(Git)
+    IF(GIT_FOUND)
+        EXECUTE_PROCESS(
+            COMMAND           ${GIT_EXECUTABLE} log --pretty=format:%h -n 1
+            WORKING_DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR}
+            OUTPUT_VARIABLE   ARBORX_GIT_COMMIT_HASH
+            )
+    ENDIF()
+ENDIF()
+MESSAGE("ArborX hash = '${ARBORX_GIT_COMMIT_HASH}'")
```

Comment:
Question:  Does this imply I need to reconfigure to update the hash?  (not complaining just wondering band making sure I understand)

Reply:
I believe so.